### ISSUE TITLE
Autofocus impedes keyboard scrolling

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/tr-pages#78]
There is also now an auto-focus into the "title" field of the search bar of the TR page. This means keyboard-based page scroll does not work, and is an accessibility problem.

**Comment by tripu** [comment]
I read that there are advantages to each approach (autofocus on main input of main form on the page, or not), but can't find a definitive answer. I understand that that autofocus may be useful for screen-readers, too.

Is there an authoritative answer about this, in terms of accessibility?
